experimental/QuadFormAndIsom/src/embeddings.jl

```diff
@@ -1025,7 +1100,7 @@ function _classes_isomorphic_subgroups(q::TorQuadModule,
     push!(blocks, compose(__f, j))
     push!(ni, ngens(domain(__f)))
   end
-  D, inj, proj = biproduct(domain.(blocks))
+  D, inj, proj = Hecke._biproduct(domain.(blocks))
```

Comment:
Since I should remove the `biproduct` function from user interface in Hecke, I instead call the internal version of it